pkg/html/html.go

```diff
@@ -754,12 +752,9 @@ func WriteLandingPage(w http.ResponseWriter, releases []string) {
 	fmt.Fprintf(w, landingHtmlPageEnd)
 }
 
-func PrintHtmlReport(w http.ResponseWriter, req *http.Request, bugCache buganalysis.BugCache, report, prevReport util.TestReport, endDay, jobTestCount int) {
+func PrintHtmlReport(w http.ResponseWriter, req *http.Request, report, prevReport util.TestReport, endDay, jobTestCount int) {
 	w.Header().Set("Content-Type", "text/html;charset=UTF-8")
 	fmt.Fprintf(w, htmlPageStart, "Release CI Health Dashboard")
-	if err := bugCache.LastUpdateError(); err != nil {
-		fmt.Fprintf(w, bugLookupWarning, err)
-	}
```

Comment:
am i missing how you're replacing this message on the page?

Reply:
> am i missing how you're replacing this message on the page?

I owe you I think. After the final type split, we can add a banner/warning list